torchvision/transforms/v2/functional/_geometry.py

```diff
@@ -451,6 +451,12 @@ def _parallelogram_to_bounding_boxes(parallelogram: torch.Tensor) -> torch.Tenso
         torch.Tensor: Tensor of same shape as input containing the rectangle coordinates.
                      The output maintains the same dtype as the input.
     """
+    dtype = parallelogram.dtype
+    acceptable_dtypes = [torch.float32]
```

Comment:
I think we should consider float64 to also be an acceptable dtype? Otherwise we risk converting a float64 into a float32, which may be unnecessary.

```suggestion
    acceptable_dtypes = [torch.float32, torch.float64]
```